Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -206,6 +207,10 @@ extension XcodeModuleProtocol  {
             }
         }
 
+        if let pkgArgs = try? self.pkgConfigArgs() { 
+            buildSettings["OTHER_CFLAGS"] = serializeArray(pkgArgs.xcc)
```

Comment:
is it okay to use this or is it better to use `OTHER_SWIFT_FLAGS` instead with arguments prefixed `-Xcc`?